Woof/Woof/Model/Walking/Location.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/**
+ The geographical location of the walking service.
+ Conforms to the `Codable` protocols.
```

Comment:
Why do we need it? I can see in code which protocols the type adopts.

Reply:
You're right, just a few lines later you can see this in the class declaration.
Deleted with 7f70e76